packages/sdk/react-client/package.json

```diff
@@ -29,15 +30,15 @@
     "@dxos/react-async": "workspace:*",
     "@dxos/rpc": "workspace:*",
     "@dxos/rpc-tunnel": "workspace:*",
+    "@dxos/text-model": "workspace:*",
     "@dxos/util": "workspace:*",
+    "@faker-js/faker": "^7.3.0",
```

Comment:
hmmm concerned about this, I'm assuming you moved all deps from dev deps that the new plugin found were required at run time? Perhaps some of these should be peer or even optional dependencies

Reply:
yes, I think a lot of the ones that were moved are only used in /testing entrypoint. The downside of moving it here is dependency bloat at install time, but they should not be included in a built bundle.

We can do a pass to clean this up and move them to peer/optional deps if necessary